packages/tauri/package.json

```diff
@@ -26,6 +28,7 @@
         "webpack": "5.52.0",
         "webpack-cli": "4.9.1",
         "webpack-dev-server": "4.7.1",
+        "@iarna/toml": "2.2.5",
```

Comment:
This was the smallest and easiest package I found to parse + generate `toml` files. It's also only used to build things, not in the built app.